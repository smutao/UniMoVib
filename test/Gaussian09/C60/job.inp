Note: The Ih group is not supported by rotational entropy in Gaussian, so the C1
symmetry has to be used instead to reproduce Gaussian's Gibbs free energy.

 $contrl
   qcprog="gaussian"
   IFConc=.t.
 $end

 $qcdata
   fchk="c60.fchk"
 $end

 $thermo
   pg="c1"
 $end
