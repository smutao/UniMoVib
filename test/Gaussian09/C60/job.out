 =======================================================================================================================
       _/     _/                _/     _/            _/     _/
      _/     _/            _/  _/|  _/_/            _/     _/  _/  _/        UniMoVib:          A unified interface for
     _/     _/   _/_/_/       _/ |_/ _/   _/_/_/   _/     _/      _/           molecular harmonic vibrational frequency
    _/     _/  _/    _/  _/  _/  _/ _/  _/    _/  _/     _/  _/  _/_/_/        calculations.
   _/     _/  _/    _/  _/  _/     _/  _/    _/  _/    _/   _/  _/   _/
  _/     _/  _/    _/  _/  _/     _/  _/    _/   _/  _/    _/  _/   _/       Ver. 1.0.0, AUG 23, 2017
   _/_/_/   _/    _/  _/  _/     _/   _/_/_/     _/_/     _/  _/_/_/         https://github.com/zorkzou/UniMoVib
 =======================================================================================================================


 QC Program:      Gaussian
 FCHK file:       c60.fchk


 Cartesian coordinates (Angstrom)
 ------------------------------------------------------------------------------------------
   No.   Atom    ZA                 X             Y             Z                Mass
 ------------------------------------------------------------------------------------------
     1    C       6            1.18821151    0.38607332    3.33957375           12.00000000
     2    C       6            2.30942676    0.75037825    2.61096390           12.00000000
     3    C       6           -2.30942676   -0.75037825   -2.61096390           12.00000000
     4    C       6           -1.18821151   -0.38607332   -3.33957375           12.00000000
     5    C       6            1.42730424   -3.21387526    0.58945775           12.00000000
     6    C       6            0.73435509   -3.43902808   -0.58945775           12.00000000
     7    C       6           -0.73435509    3.43902808    0.58945775           12.00000000
     8    C       6           -1.42730424    3.21387526   -0.58945775           12.00000000
     9    C       6           -2.61551574   -1.57844242    1.83881726           12.00000000
    10    C       6           -3.04378186   -0.26037483    1.83881726           12.00000000
    11    C       6            3.04378186    0.26037483   -1.83881726           12.00000000
    12    C       6            2.61551574    1.57844242   -1.83881726           12.00000000
    13    C       6            0.73435509   -1.01075308    3.33957375           12.00000000
    14    C       6            0.00000000    1.24935951    3.33957375           12.00000000
    15    C       6            3.04378186   -0.26037483    1.83881726           12.00000000
    16    C       6            2.30942676    1.99973776    1.83881726           12.00000000
    17    C       6           -2.30942676   -1.99973776   -1.83881726           12.00000000
    18    C       6           -3.04378186    0.26037483   -1.83881726           12.00000000
    19    C       6            0.00000000   -1.24935951   -3.33957375           12.00000000
    20    C       6           -0.73435509    1.01075308   -3.33957375           12.00000000
    21    C       6            0.69294914   -2.97526882    1.83881726           12.00000000
    22    C       6            2.61551574   -2.35058907    0.58945775           12.00000000
    23    C       6           -0.73435509   -3.43902808   -0.58945775           12.00000000
    24    C       6            1.18821151   -2.81434833   -1.83881726           12.00000000
    25    C       6           -1.18821151    2.81434833    1.83881726           12.00000000
    26    C       6            0.73435509    3.43902808    0.58945775           12.00000000
    27    C       6           -2.61551574    2.35058907   -0.58945775           12.00000000
    28    C       6           -0.69294914    2.97526882   -1.83881726           12.00000000
    29    C       6           -1.42730424   -1.96451575    2.61096390           12.00000000
    30    C       6           -2.61551574   -2.35058907    0.58945775           12.00000000
    31    C       6           -2.30942676    0.75037825    2.61096390           12.00000000
    32    C       6           -3.49763827    0.36430492    0.58945775           12.00000000
    33    C       6            3.49763827   -0.36430492   -0.58945775           12.00000000
    34    C       6            2.30942676   -0.75037825   -2.61096390           12.00000000
    35    C       6            2.61551574    2.35058907   -0.58945775           12.00000000
    36    C       6            1.42730424    1.96451575   -2.61096390           12.00000000
    37    C       6            1.42730424   -1.96451575    2.61096390           12.00000000
    38    C       6            0.00000000    2.42827500    2.61096390           12.00000000
    39    C       6            2.61551574   -1.57844242    1.83881726           12.00000000
    40    C       6            1.18821151    2.81434833    1.83881726           12.00000000
    41    C       6           -1.18821151   -2.81434833   -1.83881726           12.00000000
    42    C       6           -2.61551574    1.57844242   -1.83881726           12.00000000
    43    C       6            0.00000000   -2.42827500   -2.61096390           12.00000000
    44    C       6           -1.42730424    1.96451575   -2.61096390           12.00000000
    45    C       6           -0.69294914   -2.97526882    1.83881726           12.00000000
    46    C       6            3.04378186   -1.76113132   -0.58945775           12.00000000
    47    C       6           -1.42730424   -3.21387526    0.58945775           12.00000000
    48    C       6            2.30942676   -1.99973776   -1.83881726           12.00000000
    49    C       6           -2.30942676    1.99973776    1.83881726           12.00000000
    50    C       6            1.42730424    3.21387526   -0.58945775           12.00000000
    51    C       6           -3.04378186    1.76113132    0.58945775           12.00000000
    52    C       6            0.69294914    2.97526882   -1.83881726           12.00000000
    53    C       6           -0.73435509   -1.01075308    3.33957375           12.00000000
    54    C       6           -3.04378186   -1.76113132   -0.58945775           12.00000000
    55    C       6           -1.18821151    0.38607332    3.33957375           12.00000000
    56    C       6           -3.49763827   -0.36430492   -0.58945775           12.00000000
    57    C       6            3.49763827    0.36430492    0.58945775           12.00000000
    58    C       6            1.18821151   -0.38607332   -3.33957375           12.00000000
    59    C       6            3.04378186    1.76113132    0.58945775           12.00000000
    60    C       6            0.73435509    1.01075308   -3.33957375           12.00000000
 ------------------------------------------------------------------------------------------


 <<< SYMMETRY >>>
 Molecular Point Group (Z)   :   Ih  
 Molecular Point Group (Z+M) :   Ih  


 ************************************
 ***  Properties of Normal Modes  ***
 ************************************

 Results of vibrations:
 Normal frequencies (cm**-1), reduced masses (AMU),
 force constants (mDyn/A), IR intensities (km/mol)

                             1         2         3         4         5         6         7         8         9        10
          Irreps            Hg        Hg        Hg        Hg        Hg       T2u       T2u       T2u        Gu        Gu
     Frequencies        251.22    251.22    251.22    251.22    251.22    332.18    332.18    332.18    343.45    343.45
 Force constants        0.4462    0.4462    0.4462    0.4462    0.4462    0.7802    0.7802    0.7802    0.8340    0.8340
  IR intensities        0.0000    0.0000    0.0000    0.0000    0.0000    0.0000    0.0000    0.0000    0.0000    0.0000

                            11        12        13        14        15        16        17        18        19        20
          Irreps            Gu        Gu        Hu        Hu        Hu        Hu        Hu        Hg        Hg        Hg
     Frequencies        343.45    343.45    383.93    383.93    383.93    383.93    383.93    423.13    423.13    423.13
 Force constants        0.8340    0.8340    1.0422    1.0422    1.0422    1.0422    1.0422    1.2659    1.2659    1.2659
  IR intensities        0.0000    0.0000    0.0000    0.0000    0.0000    0.0000    0.0000    0.0000    0.0000    0.0000

                            21        22        23        24        25        26        27        28        29        30
          Irreps            Hg        Hg        Gg        Gg        Gg        Gg        Hu        Hu        Hu        Hu
     Frequencies        423.13    423.13    467.88    467.88    467.88    467.88    519.33    519.33    519.33    519.33
 Force constants        1.2659    1.2659    1.5478    1.5478    1.5478    1.5478    1.9068    1.9068    1.9068    1.9068
  IR intensities        0.0000    0.0000    0.0000    0.0000    0.0000    0.0000    0.0000    0.0000    0.0000    0.0000

                            31        32        33        34        35        36        37        38        39        40
          Irreps            Hu       T1u       T1u       T1u        Gg        Gg        Gg        Gg       T2g       T2g
     Frequencies        519.33    537.79    537.79    537.79    548.96    548.96    548.96    548.96    553.59    553.59
 Force constants        1.9068    2.0448    2.0448    2.0448    2.1306    2.1306    2.1306    2.1306    2.1667    2.1667
  IR intensities        0.0000    5.2129    5.2130    5.2128    0.0000    0.0000    0.0000    0.0000    0.0000    0.0000

                            41        42        43        44        45        46        47        48        49        50
          Irreps           T2g       T1g       T1g       T1g        Ag        Hu        Hu        Hu        Hu        Hu
     Frequencies        553.59    583.83    583.83    583.83    595.40    661.07    661.07    661.07    661.07    661.07
 Force constants        2.1667    2.4099    2.4099    2.4100    2.5064    3.0898    3.0898    3.0898    3.0898    3.0898
  IR intensities        0.0000    0.0000    0.0000    0.0000    0.0000    0.0000    0.0000    0.0000    0.0000    0.0000

                            51        52        53        54        55        56        57        58        59        60
          Irreps           T1u       T1u       T1u        Gu        Gu        Gu        Gu        Hg        Hg        Hg
     Frequencies        700.09    700.09    700.09    710.99    710.99    710.99    710.99    719.06    719.06    719.06
 Force constants        3.4653    3.4653    3.4653    3.5740    3.5740    3.5740    3.5740    3.6556    3.6556    3.6556
  IR intensities        5.2734    5.2749    5.2751    0.0000    0.0000    0.0000    0.0000    0.0000    0.0000    0.0000

                            61        62        63        64        65        66        67        68        69        70
          Irreps            Hg        Hg       T2u       T2u       T2u       T2g       T2g       T2g        Hu        Hu
     Frequencies        719.06    719.06    720.52    720.52    720.52    739.02    739.02    739.02    760.00    760.00
 Force constants        3.6556    3.6556    3.6704    3.6704    3.6704    3.8614    3.8614    3.8614    4.0837    4.0837
  IR intensities        0.0000    0.0000    0.0000    0.0000    0.0000    0.0000    0.0000    0.0000    0.0000    0.0000

                            71        72        73        74        75        76        77        78        79        80
          Irreps            Hu        Hu        Hu        Gg        Gg        Gg        Gg       T1g       T1g       T1g
     Frequencies        760.00    760.00    760.00    784.33    784.33    784.33    784.33    807.68    807.68    807.68
 Force constants        4.0837    4.0837    4.0837    4.3494    4.3494    4.3494    4.3494    4.6122    4.6122    4.6122
  IR intensities        0.0000    0.0000    0.0000    0.0000    0.0000    0.0000    0.0000    0.0000    0.0000    0.0000

                            81        82        83        84        85        86        87        88        89        90
          Irreps            Gu        Gu        Gu        Gu       T2g       T2g       T2g        Hg        Hg        Hg
     Frequencies        825.48    825.48    825.48    825.48    825.77    825.77    825.77    893.19    893.19    893.19
 Force constants        4.8177    4.8177    4.8177    4.8177    4.8212    4.8212    4.8212    5.6406    5.6406    5.6406
  IR intensities        0.0000    0.0000    0.0000    0.0000    0.0000    0.0000    0.0000    0.0000    0.0000    0.0000

                            91        92        93        94        95        96        97        98        99       100
          Irreps            Hg        Hg        Au        Gu        Gu        Gu        Gu       T2u       T2u       T2u
     Frequencies        893.20    893.20    904.03   1057.64   1057.65   1057.65   1057.65   1082.18   1082.18   1082.18
 Force constants        5.6406    5.6406    5.7782    7.9088    7.9088    7.9089    7.9089    8.2800    8.2800    8.2800
  IR intensities        0.0000    0.0000    0.0000    0.0000    0.0000    0.0000    0.0000    0.0000    0.0000    0.0000

                           101       102       103       104       105       106       107       108       109       110
          Irreps            Gg        Gg        Gg        Gg        Hg        Hg        Hg        Hg        Hg       T2u
     Frequencies       1152.54   1152.54   1152.54   1152.54   1197.52   1197.52   1197.52   1197.52   1197.52   1226.55
 Force constants        9.3917    9.3917    9.3917    9.3917   10.1390   10.1390   10.1390   10.1390   10.1390   10.6365
  IR intensities        0.0000    0.0000    0.0000    0.0000    0.0000    0.0000    0.0000    0.0000    0.0000    0.0000

                           111       112       113       114       115       116       117       118       119       120
          Irreps           T2u       T2u        Hu        Hu        Hu        Hu        Hu       T1u       T1u       T1u
     Frequencies       1226.55   1226.55   1265.42   1265.42   1265.42   1265.42   1265.42   1272.54   1272.54   1272.54
 Force constants       10.6365   10.6365   11.3214   11.3214   11.3214   11.3214   11.3214   11.4491   11.4491   11.4492
  IR intensities        0.0000    0.0000    0.0000    0.0000    0.0000    0.0000    0.0000   16.7893   16.7856   16.7828

                           121       122       123       124       125       126       127       128       129       130
          Irreps            Hg        Hg        Hg        Hg        Hg        Gg        Gg        Gg        Gg       T1g
     Frequencies       1306.53   1306.53   1306.54   1306.54   1306.55   1317.52   1317.52   1317.52   1317.52   1320.16
 Force constants       12.0690   12.0691   12.0691   12.0692   12.0693   12.2728   12.2728   12.2728   12.2729   12.3220
  IR intensities        0.0000    0.0000    0.0000    0.0000    0.0000    0.0000    0.0000    0.0000    0.0000    0.0000

                           131       132       133       134       135       136       137       138       139       140
          Irreps           T1g       T1g        Gu        Gu        Gu        Gu       T2g       T2g       T2g        Hu
     Frequencies       1320.16   1320.16   1331.02   1331.02   1331.03   1331.03   1354.48   1354.48   1354.48   1356.89
 Force constants       12.3220   12.3220   12.5256   12.5257   12.5258   12.5259   12.9712   12.9712   12.9712   13.0174
  IR intensities        0.0000    0.0000    0.0000    0.0000    0.0000    0.0000    0.0000    0.0000    0.0000    0.0000

                           141       142       143       144       145       146       147       148       149       150
          Irreps            Hu        Hu        Hu        Hu        Gu        Gu        Gu        Gu        Hg        Hg
     Frequencies       1356.90   1356.90   1356.90   1356.90   1507.46   1507.47   1507.47   1507.47   1527.69   1527.69
 Force constants       13.0174   13.0174   13.0174   13.0174   16.0666   16.0667   16.0668   16.0669   16.5007   16.5007
  IR intensities        0.0000    0.0000    0.0000    0.0000    0.0000    0.0000    0.0000    0.0000    0.0000    0.0000

                           151       152       153       154       155       156       157       158       159       160
          Irreps            Hg        Hg        Hg       T1u       T1u       T1u        Gg        Gg        Gg        Gg
     Frequencies       1527.69   1527.70   1527.70   1570.01   1570.01   1570.01   1585.93   1585.93   1585.93   1585.94
 Force constants       16.5008   16.5008   16.5009   17.4276   17.4276   17.4276   17.7828   17.7829   17.7829   17.7829
  IR intensities        0.0000    0.0000    0.0000   27.5843   27.5858   27.5860    0.0000    0.0000    0.0000    0.0000

                           161       162       163       164       165       166       167       168       169       170
          Irreps            Ag       T2u       T2u       T2u        Hu        Hu        Hu        Hu        Hu        Hg
     Frequencies       1607.43   1641.32   1641.32   1641.32   1643.05   1643.05   1643.05   1643.05   1643.05   1662.52
 Force constants       18.2683   19.0466   19.0467   19.0467   19.0868   19.0868   19.0868   19.0868   19.0868   19.5418
  IR intensities        0.0000    0.0000    0.0000    0.0000    0.0000    0.0000    0.0000    0.0000    0.0000    0.0000

                           171       172       173       174
          Irreps            Hg        Hg        Hg        Hg
     Frequencies       1662.52   1662.52   1662.52   1662.52
 Force constants       19.5418   19.5418   19.5418   19.5418
  IR intensities        0.0000    0.0000    0.0000    0.0000


 *********************************************
 ***   Thermal Contributions to Energies   ***
 *********************************************

 Temperature               :       298.150000    Kelvin
 Pressure                  :         1.000000    Atm
 Molecular mass            :       720.000000    AMU
 Elec. total energy        :         0.000000    Hartree
 Scale factor of Frequency :         1.000000
 Rot. symmetry number      :         1
 The C1   point group is used to calculate rotational entropy.

 Principal axes and moments of inertia in atomic units:
                                     1                   2                   3
     Eigenvalues --              21792.629591        21792.629596        21792.629599
           X                         0.000000            1.000000            0.000000
           Y                        -0.216521            0.000000            0.976278
           Z                        -0.976278            0.000000           -0.216521

 Rotational temperatures             0.003974            0.003974            0.003974    Kelvin
 Rot. constants A, B, C              0.002762            0.002762            0.002762    cm**-1
                                     0.082814            0.082814            0.082814    GHz

 Thermal correction energies                              Hartree            kcal/mol
 Zero-point Energy                          :            0.383406          240.590788
 Thermal correction to Energy               :            0.404225          253.655255
 Thermal correction to Enthalpy             :            0.405170          254.247741
 Thermal correction to Gibbs Free Energy    :            0.339999          213.352295


 Job terminated correctly! Wed Aug 23 19:04:54 2017
